modules/ROOT/pages/runtime-fabric-deployment-mmp-reference.adoc

```diff
@@ -49,6 +50,17 @@ For example:
 
 This will set 100MB of memory to each replica.
 
+| memoryMax| Amount of max memory to be allocated for each replica of the application. If memoryReserved configuration is present, ensure that this value is equal or higher. +
+
+For example:
+
+<deploymentSettings> +
+    <memoryMax>200Mi</memoryMax> +
```

Comment:
Is this correct? Shouldn't it be `Mb` or `M`

Reply:
Yes, Mi is correct